### PR TITLE
[many ports]Add vcpkg_fixup_pkgconfig

### LINKED ISSUE #19720: [vcpkg_fixup_pkgconfig] Fails to fix up `prefix=${CMAKE_CURRENT_PACKAGES_DIR}`
See epoxy.pc for example:

```pc
prefix=/Users/nimazzuc/projects/vcpkg/packages/libepoxy_arm64-osx # <- this is incorrect
libdir=${prefix}/lib
includedir=${prefix}/include

epoxy_has_glx=0
epoxy_has_egl=0
epoxy_has_wgl=0

Name: epoxy
Description: GL dispatch library
Version: 1.5.5
Libs: -L${libdir} -lepoxy -ldl
Cflags: -I${includedir}
```

### PR DESCRIPTION
**Pull request by JackBoosY**
Fixes #19720.

### CODE REVIEW

**Review by strega-nil — APPROVED** [review]

**Review by strega-nil-ms — APPROVED** [review]

**Comment by strega-nil-ms** [comment]
Thanks Jack!

File: ports/libepoxy/vcpkg.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "libepoxy",
+  "version-string": "1.5.5",
```
Comment (ras0219-msft) [inline ports/libepoxy/vcpkg.json:3]:
```suggestion
  "version-semver": "1.5.5",
```

**Review by ras0219-msft — CHANGES_REQUESTED** [review]

File: versions/l-/libepoxy.json [hunk 2]
```diff
@@ -1,5 +1,10 @@
 {
   "versions": [
+    {
+      "git-tree": "8d639774e0732def8989992a3b2d89bcdfb768e3",
+      "version-string": "1.5.5",
```
Comment (NancyLi1013) [inline versions/l-/libepoxy.json:5]:
```suggestion
      "version-semver": "1.5.5",
```

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

File: versions/l-/libepoxy.json [hunk 3]
```diff
@@ -1,5 +1,10 @@
 {
   "versions": [
+    {
+      "git-tree": "8d639774e0732def8989992a3b2d89bcdfb768e3",
```
Comment (NancyLi1013) [inline versions/l-/libepoxy.json:4]:
```suggestion
      "git-tree": "66e28966117a23b6e9ac6cba7e71ecad8a2803ea",
```

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

**Review by NancyLi1013 — APPROVED** [review]

**Pull request merged**